dissect/target/plugins/browsers/firefox.py

```diff
@@ -1,19 +1,21 @@
+import json
+from typing import Iterator
+
+from flow.record.fieldtypes import datetime, path
+
 from dissect.sql import sqlite3
 from dissect.sql.exceptions import Error as SQLError
-from dissect.util.ts import from_unix_us
-
+from dissect.sql.sqlite3 import Row, SQLite3
 from dissect.target.exceptions import FileNotFoundError, UnsupportedPluginError
 from dissect.target.helpers.descriptor_extensions import UserRecordDescriptorExtension
 from dissect.target.helpers.record import create_extended_descriptor
 from dissect.target.plugin import Plugin, export
 from dissect.target.plugins.browsers.browser import (
+    GENERIC_DOWNLOAD_RECORD_FIELDS,
     GENERIC_HISTORY_RECORD_FIELDS,
     try_idna,
 )
-
-FirefoxBrowserHistoryRecord = create_extended_descriptor([UserRecordDescriptorExtension])(
```

Comment:
Can you revert this change? The `HISTORY_RECORD` constant is only relevant for anything that uses the `ChromiumMixin`, anything else should still use this style of defining record descriptors.

Reply:
This also applies to the newly added downloads record.